src/improvements/tasks/sep/025-opcm-upgrade-v410-freya-bn-u15/config.toml

```diff
@@ -0,0 +1,24 @@
+# Freya Devnet on u15
+l2chains = [
+    {name = "freya-u15-0", chainId = 420110013},
+    {name = "freya-u15-1", chainId = 420110014}
+]
+
+fallbackAddressesJsonPath = "src/improvements/tasks/sep/025-opcm-upgrade-v410-freya-bn-u15/addresses.json"
+templateName = "OPCMUpgradeV410"
+
+[[opcmUpgrades]]
+chainId = 420110013
+# Freya prestate, to get:
+absolutePrestate = "0x039fb00f964358d326de5804afb4e70a1ca34f9e56c60350b6c8a41fff302490"
```

Comment:
Are you sure this is the right prestate? In the [prestate reviews](https://www.notion.so/oplabs/U16a-25bf153ee16280aa9050e03ddb4bbaf5?v=1bdf153ee1628012962d000c0011a34a&source=copy_link) page I see 0x03eb07101fbdeaf3f04d9fb76526362c1eea2824e4c6e970bdb19675b72e4fc8, cc @Inphi